apps/myaccount/src/components/profile/profile.tsx

```diff
@@ -188,6 +198,21 @@ export const Profile: FunctionComponent<ProfileProps> = (props: ProfileProps): R
                 } else {
                     if (schemaNames[0] === "name") {
                         tempProfileInfo.set(schema.name, profileDetails.profileInfo[schemaNames[0]][schemaNames[1]]);
+
+                    } else if (isCanonical) {
+                        let indexOfType = -1;
+                        profileDetails?.profileInfo[schemaNamesCanonicalType[0]]?.forEach((canonical: any) => {
```

Comment:
Can we create an interface for this instead of using `any`?

Reply:
Fixed with 2fec35b068334696276f45f5b8045f98bd5c8151